### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by rnmy**

### CODE REVIEW

**Comment by rnmy** [comment]
Should we also mention that the name provided for adding consultations is independent of the student list so that testers will not be expecting the details in consultations to change with edits to the student list :")

File: docs/UserGuide.md [hunk 1]
```diff
@@ -97,6 +97,8 @@ Examples:
 
 Shows a list of all students in charge by the TA.
 
+You may navigate to the Students tab to view the full list of students.
```
Comment (chrisgzf) [inline docs/UserGuide.md:100]:
```suggestion
You may click on the Students tab to view the full list of students.
```
Perhaps this phrasing will be less vague?

**Review by kwannoel — APPROVED** [review]

File: docs/UserGuide.md [hunk 2]
```diff
@@ -230,36 +232,34 @@ Examples:
...
 ### Listing all consultations: `list-consult`
 
-Shows a list of all consultations in charge by the TA.
-
-Format: `list-consult`
-
-### Exiting the program : `exit`
-
-Exits the program.
+Shows a list of all consultations in charge by the TA.
```
Comment (chrisgzf) [inline docs/UserGuide.md:235]:
```suggestion
Shows a list of all consultations that has been arranged with your students. 
```

**Review by chrisgzf — COMMENTED** [review]
Finishing up in another PR

Edit: Changes tracked in https://github.com/AY2021S1-CS2103T-F11-1/tp/pull/204

**Pull request merged**